GNUmakefile

```diff
@@ -41,13 +41,12 @@ cov:
 	gocov test ${GOFILES} | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test:
+test: dev
 	@./scripts/verify_no_uuid.sh
```

Comment:
This `verify_no_uuid.sh` script looks like it has gone stale (needs to check all of the files under consul/state), and is kind of a brittle check so I think we can delete it. It was trying to make sure that nobody ever generates a UUID inside the state store on the far side of a Raft apply, but I think we need to ensure that through code review since there are lots of ways to create non-deterministic things.